techlibs/ice40/synth_ice40.cc

```diff
@@ -37,6 +37,10 @@ struct SynthIce40Pass : public ScriptPass
 		log("\n");
 		log("This command runs synthesis for iCE40 FPGAs.\n");
 		log("\n");
+		log("    -device < hx | lp | u >\n");
```

Comment:
Perhaps we should mention this is only for abc9 flows? Otherwise the lack of a difference with the default flow might be confusing.